schema.rst

```diff
@@ -264,6 +298,26 @@ even for Z > Y
 (with the understanding that, in such cases, the tool
 may miss non-critical information that the CPS provided).
 
+:attribute:`Deduplicate`
+------------------------
+
+:Type: |bool| :separator:`or` |string| (restricted)
+:Applies To: |envvar|
+:Required: No
+
+Specifies whether duplicate values
+in the environment variable should be removed.
+(Values are separated by the specified `Separator`_.)
+
+If enabled, deduplication takes place *after*
+the environment variable has been modified.
+If set to :string:`keep-last`,
+only the right-most value is retained.
+Otherwise (|true| or :string:`keep-first`)
+only the left-most value is retained.
+
+The default is |false|.
```

Comment:
Most tools I'm familiar with keep the last definition of something, not the first. Python's argparse works this way, bash and zsh do this with env (so do Python, Rust, and C/C++) vars (`export FOO="1" FOO="2"; echo $FOO` will print `2`), so I would expect `true == 'keep-last'`, rather than `keep-first`? But maybe it makes sense to just have this be a union of strings `none | keep-first | keep-last` with a default of `none`, since that's simpler and less likely to be misinterpreted?